src/lib.rs

```diff
@@ -0,0 +1,105 @@
+//! Tasks defined rust_actix-basic.pdf
```

Comment:
So, knowing this isn't a library, rename this file to something that makes more sense.  You can even take it further to add more files and have more separation of concerns among files.